.github/workflows/ci.yml

```
@@ -10,15 +10,16 @@ on:
 jobs:
 
   build:
-    name: Build ${{ matrix.package }}
-    runs-on: ubuntu-latest
+    name: ${{ matrix.package.job_description }} ${{ matrix.package.name }} on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
 
     strategy:
       matrix:
-        include:
-          - package: flox
-          - package: flox-bash
-          - package: nix-editor
+        package:
+        - {name: flox, job_description: "Build and test"}
+        - {name: flox-bash, job_description: "Build" }
+        - {name: nix-editor, job_description: "Build" }
+        os: [ubuntu-latest, macos-latest]
```

```suggestion
        os: 
        - ubuntu-latest
        - macos-latest
```
Just nitpicking, but it'd be more consistent with the others.